shop/src/main/java/ro/msg/learning/shop/mapper/CustomerMapper.java

```diff
@@ -0,0 +1,16 @@
+package ro.msg.learning.shop.mapper;
+
+import org.springframework.stereotype.Component;
+import ro.msg.learning.shop.domain.Customer;
+import ro.msg.learning.shop.dto.CustomerDto;
+
+@Component
+public class CustomerMapper {
+    public CustomerDto toCustomerDto(Customer customer){
+        return new CustomerDto(customer.getId(),customer.getFirstName(),customer.getLastName(),customer.getUsername(),customer.getPassword(),customer.getEmailAddress());
+    }
+
```

Comment:
You don't have to modify now the mappers, but in the future consider using Builder, the code looks prettier when creating objects.